runhouse/rns/rns_client.py

```diff
@@ -21,7 +22,12 @@
     ResourceAccess,
 )
 
-from runhouse.utils import locate_working_dir
+from runhouse.utils import locate_working_dir, sync_function
+
+
+class UpdateDenResponse(BaseModel):
```

Comment:
maybe we just reuse the existing `Response` we already have?

Reply:
I'm thinking to remove those `UpdateDenResponse` and `ResourceStatusData` at all, since we have validation of the status schema on the den side. wdyt?

Reply:
i think `ResourceStatusData` is still helpful on the OSS side to ensure we are constructing it properly

Reply:
btw - tried to reuse `Response`, but pydentic throws an error that  Response can't be used as a value type in a  model (makes sense).